docs/reference/commandline/run.md

```diff
@@ -262,6 +262,13 @@ https://docs.docker.com/engine/installation/binaries/#/get-the-linux-binary)),
 you give the container the full access to create and manipulate the host's
 Docker daemon.
 
+On Windows, the paths must be specified using Windows-style semantics. Windows
+containers do not support the target of a volume or bind-mount being pre-
+populated in the container image.
```

Comment:
> Is that going to be supported?

We know it's a deficiency, has been forever (well TP4 when the functionality was introduced), but fixing it is hard by design of the mapped directories in the platform. 

> What is the use of a bind-mounted directory in that case, that's a big difference in semantics

You can still use bind-mounts just fine if the target doesn't exist. That's the caveat.